AAI/AAIConnectProfile.md

```diff
@@ -426,6 +431,8 @@ the Broker.
                 any other Broker involved in the propagation of the claims to
                 also be trusted if the Claim Clearinghouse needs to restrict its
                 trust model).
+            2.  If Embedded Token contains `aud` field, Clearinghouse MUST check
+                that the Embedded Token's `aud` contains `iss` of access token provided by Broker.
```

Comment:
Couple of things here:
 - contains is too vague, it must be an exact match as per https://tools.ietf.org/html/rfc3986#section-6.2.1
 - "access token provided by Broker" can be ambiguous in cases where the there are multiple brokers involved or multiple embedded tokens that happen to be signed by one or more brokers. The intent comes across, but it just isn't tightly worded.

To fix:
"[matches] (https://tools.ietf.org/html/rfc3986#section-6.2.1) the Broker's `iss` claim (i.e. a Broker's access token `iss` claim must match the `aud` claim within its Embedded Tokens if the Embedded Token `aud` claim is provided)"